documentation/pom.xml

```diff
@@ -1,103 +1,103 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There were windows line endings in this pom and also whitespaces inconsistencies...